### ISSUE TITLE
entc generate failed

### ISSUE THREAD

**Issue by freezeChen** [facebookincubator/ent#70]
user.go
```
package schema

import (
	"github.com/facebookincubator/ent"
	"github.com/facebookincubator/ent/schema/edge"
	"github.com/facebookincubator/ent/schema/field"
)

// User holds the schema definition for the User entity.
type User struct {
	ent.Schema
}

// Fields of the User.
func (User) Fields() []ent.Field {
	return []ent.Field{
		field.Int("age"),
		field.String("name"),
	}
}

// Edges of the User.
func (User) Edges() []ent.Edge {
	return []ent.Edge{
		edge.To("friends", User.Type),
	}
}

```

run entc generate ./ent/schema   is failed,only generate an empty folder. 
```
$ entc generate ./ent/schema
entc/gen: write file C:\project\gowork\src\awesomeProject\ent\user_create.go: formatting source: C:\project\gowork\src\awesomeProject\e
nt\user_create.go:21:19: unknown escape sequence (and 1 more errors)
```

**Comment by alexsn** [comment]
Don't think we ever tested `entc` on windows. I'll take a look.

**Comment by freezeChen** [comment]
in /entc/cmd/entc/entc.go  function  loadGraph   
```
cfg.Package = filepath.Dir(spec.PkgPath)
```
when os is windows,  package is failed

**Comment by a8m** [comment]
Closed with 7597f079129814c3ac37f823fae9d1b6f862e40b.

Thanks for reporting this issue.
